lang/cs/.nuget/packages.config

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like those lines define default NuGet source. Not sure why the XUnit doesn't want it?